class-backend.php

```diff
@@ -1105,7 +1106,7 @@ function metabox_content_woocommernfe_nfe_emitida( $order ) {
 	function metabox_content_woocommernfe_informacoes_adicionais( $order ) {
 
 		// Vars
-		if (isset($order->ID)){
+		if ( ! is_a( $order, 'WC_Order' ) && isset($order->ID) ) {
```

Comment:
Na versão do WooCommerce que estou usando (9.1.2) este trecho gera um alerta de que não se deve acessar a propriedade ID diretamente. 
No meu ambiente de testes, com HPOS ativado e modo de compatibilidade desativado, não consegui reproduzir o caso em que o parâmetro $order não fosse um objeto do tipo WC_Order e que portanto seria necessário fazer uma chamada para wc_get_order().

Pensei que talvez esse trecho do código tenha sido adicionado por uma questão de retrocompatibilidade, então adicionei uma nova condição no if para que ele somente acesse o $order->ID se a variável $order não for da classe WC_Order.